vignettes/irecist.Rmd

```diff
@@ -0,0 +1,858 @@
+---
+title: "Creating ADRS with iRECIST endpoints"
```

Comment:
"iRescist response criteria" instead of "iRecisrt endpoints"?

Reply:
keep as is